src/backend/gitalytics_api/enums.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+class CookieKey(Enum):
```

Comment:
nenn mal um zu `CookieKeys`. (war gerade sehr in `auth/github.py`z122 sehr verwirrt

Reply:
Oh, das ist nicht gut.
Eigentlich kenne ich es so, dass man enums immer im Singular schreibt. Aus einem ähnlichen Grund, dass man DB Tabellennamen immer im Singular schreibt. Man hat ja immer ein Objekt von dem Typ <Enum>, genau wie man immer einen Klassennamen im Singular schreibt.

Nach einer kleinen google suche, scheint das auch der Standard zu sein. Glaubst du du könntest dich damit mit etwas Zeit anfreunden? : )